### DISCUSSION TITLE
Very low kit volume

### DISCUSSION THREAD

**Discussion by Qui-Gon-JooL** [SynthstromAudible/DelugeFirmware#1647] [Q&A] [answered]
Is it normal that since 3 nightly update, as well as on the last, the sound of the kits has become very weak, on my old tracks I can hardly hear them anymore, and on a new tracks i have to put all the kit volume parameters, line and the velocity at maximum to hear them ?

  Is this the expected behavior ?

**Comment by sapphire-arches** [comment, marked as answer]
Very low volume should be fixed in the latest nightly (7aa29dbd218994932426c19460cbfc29757aa5fa) and beta, this was an unintended side-effect of fixing the compressor so it actually compresses. Please follow #1638  for further compressor volume bugs.

**Reply by Qui-Gon-JooL** [reply]
Ok thank's.
